libevmasm/RuleList.h

```diff
@@ -174,6 +174,18 @@ std::vector<SimplificationRule<Pattern>> simplificationRuleList(
 		});
 	}
 
+	for (auto const& op: std::vector<Instruction>{
+		Instruction::ADDRESS,
+		Instruction::CALLER,
+		Instruction::ORIGIN,
+		Instruction::COINBASE
+	})
+		rules.push_back({
+			{Instruction::AND, {{u256("0xffffffffffffffffffffffffffffffffffffffff"), op}}},
```

Comment:
Should this instead be `(u256(1) << 160) - 1`?

Reply:
probably better, yes